--- CTL-ctl-1.5.2/lib/IlmCtlMath/CMakeLists.txt.orig	2023-03-14 18:23:20.790979681 +0100
+++ CTL-ctl-1.5.2/lib/IlmCtlMath/CMakeLists.txt	2023-03-14 18:44:50.993990051 +0100
@@ -6,7 +6,7 @@ add_library( IlmCtlMath ${DO_SHARED}
   CtlRbfInterpolator.cpp
 )
 
-target_link_libraries( IlmCtlMath IlmCtl )
+target_link_libraries( IlmCtlMath IlmCtl Imath )
 
 set_target_properties( IlmCtlMath PROPERTIES
   VERSION ${CTL_VERSION}
