--- ctl-1.4.1/IlmCtl/CtlLex.cpp.orig	2007-03-20 18:48:40.000000000 +0100
+++ ctl-1.4.1/IlmCtl/CtlLex.cpp	2013-03-28 18:59:30.520355166 +0100
@@ -57,6 +57,7 @@
 #include <CtlLContext.h>
 #include <iomanip>
 #include <cassert>
+#include <cstdlib>
 
 
 #if 0
--- ctl-1.4.1/IlmCtlSimd/CtlSimdReg.h.orig	2006-11-30 02:46:44.000000000 +0100
+++ ctl-1.4.1/IlmCtlSimd/CtlSimdReg.h	2013-03-28 19:05:40.243672986 +0100
@@ -49,6 +49,7 @@
 #ifndef INCLUDED_CTL_SIMD_REG_H
 #define INCLUDED_CTL_SIMD_REG_H
 
+#include <cstring>
 #include <typeinfo>
 #include <CtlExc.h>
 #include <Iex.h>
