--- CTL-ctl-1.5.2/lib/IlmCtlSimd/CtlSimdInst.cpp.orig	2014-06-03 03:11:24.000000000 +0200
+++ CTL-ctl-1.5.2/lib/IlmCtlSimd/CtlSimdInst.cpp	2022-02-05 08:36:14.965330834 +0100
@@ -189,7 +189,7 @@ SimdInst::executePath (SimdBoolMask &mas
 	    REPLACE_EXC
 		(e, "\n" <<
 		 xcontext.fileName() << ":" <<
-		 inst->lineNumber() << ": " << e);
+		 inst->lineNumber() << ": " << e.message());
 
 	    throw e;
 	}
