--- CTL-ctl-1.5/lib/IlmImfCtl/CMakeLists.txt.orig	2013-10-22 19:39:04.000000000 +0200
+++ CTL-ctl-1.5/lib/IlmImfCtl/CMakeLists.txt	2014-04-06 08:31:55.806391615 +0200
@@ -13,4 +13,4 @@
 
 export( TARGETS IlmImfCtl IlmCtl FILE "${PROJECT_BINARY_DIR}/CTLLibraryDepends.cmake" )
 
-install( TARGETS IlmImfCtl DESTINATION lib )
+install( TARGETS IlmImfCtl DESTINATION lib${LIB_SUFFIX} )
--- CTL-ctl-1.5/lib/IlmCtl/CMakeLists.txt.orig	2013-10-22 19:39:04.000000000 +0200
+++ CTL-ctl-1.5/lib/IlmCtl/CMakeLists.txt	2014-04-06 08:32:32.226390855 +0200
@@ -50,4 +50,4 @@
 	CtlVersion.h
  DESTINATION include/CTL )
 
-install( TARGETS IlmCtl DESTINATION lib )
+install( TARGETS IlmCtl DESTINATION lib${LIB_SUFFIX} )
--- CTL-ctl-1.5/lib/IlmCtlMath/CMakeLists.txt.orig	2013-10-22 19:39:04.000000000 +0200
+++ CTL-ctl-1.5/lib/IlmCtlMath/CMakeLists.txt	2014-04-06 08:32:50.143057151 +0200
@@ -20,6 +20,6 @@
   CtlSparseMatrix.h
  DESTINATION include/CTL )
 
-install( TARGETS IlmCtlMath DESTINATION lib )
+install( TARGETS IlmCtlMath DESTINATION lib${LIB_SUFFIX} )
 
 # TODO Add Imath and Iex libadd dependency and version info
--- CTL-ctl-1.5/lib/IlmCtlSimd/CMakeLists.txt.orig	2013-10-22 19:39:04.000000000 +0200
+++ CTL-ctl-1.5/lib/IlmCtlSimd/CMakeLists.txt	2014-04-06 08:33:07.246390126 +0200
@@ -42,4 +42,4 @@
 
 install( FILES CtlSimdInterpreter.h DESTINATION include/CTL )
 
-install( TARGETS IlmCtlSimd DESTINATION lib )
+install( TARGETS IlmCtlSimd DESTINATION lib${LIB_SUFFIX} )
--- CTL-ctl-1.5/CMakeLists.txt.orig	2013-10-22 19:39:04.000000000 +0200
+++ CTL-ctl-1.5/CMakeLists.txt	2014-04-06 08:35:26.706387222 +0200
@@ -77,7 +77,7 @@
 
 if ( PKG_CONFIG_FOUND )
 configure_file(config/CTL.pc.in "${PROJECT_BINARY_DIR}/CTL.pc" @ONLY)
-install( FILES "${PROJECT_BINARY_DIR}/CTL.pc" DESTINATION lib/pkgconfig COMPONENT dev )
+install( FILES "${PROJECT_BINARY_DIR}/CTL.pc" DESTINATION lib${LIB_SUFFIX}/pkgconfig COMPONENT dev )
 endif()
 
 install( FILES
